go/src/koding/klient/terminal/terminal.go

```diff
@@ -253,7 +255,13 @@ func (t *Terminal) Connect(r *kite.Request) (interface{}, error) {
 	}
 
 	args = append(args, command.Args...)
-	cmd := exec.Command("/usr/bin/sudo", args...)
+	var cmd *exec.Cmd
+
+	if _, err := exec.LookPath("sudo"); err == nil {
```

Comment:
1) you might cache the sudo path
2) you could use the return value from LookPath `exec.Command("/usr/bin/sudo", args...)`